examples/Cassie/cassie_utils.h

```diff
@@ -41,10 +41,15 @@ using drake::systems::lcm::LcmPublisherSystem;
 /// @param scene_graph a pointer to the SceneGraph--may be nullptr (or omitted)
 /// @param filename the URDF or SDF file to use for Cassie
 ///        omit to use default value
+/// @param add_leaf_springs Whether or not to add the 4 leaf springs in the legs
+///     Default = true
+/// @param add_loop_closure_springs Whether or not to add the loop closure
+///     distance constraint via stiff springs. Default = true.
 void addCassieMultibody(drake::multibody::MultibodyPlant<double>* plant,
     drake::geometry::SceneGraph<double>* scene_graph = nullptr,
     bool floating_base = true,
-    std::string filename = "examples/Cassie/urdf/cassie_v2.urdf");
+    std::string filename = "examples/Cassie/urdf/cassie_v2.urdf",
+    bool add_leaf_springs = true,  bool add_loop_closure_springs = true);
```

Comment:
Should we throw an error if 
filename = "examples/Cassie/urdf/cassie_fixed_spring.urdf
and 
add_leaf_springs = true?

Reply:
At the moment, if you try that, the program will crash with the (unhelpful) error message `std::bad_cast` when this code
```
 plant->AddForceElement<RevoluteSpring>(
        dynamic_cast<const drake::multibody::RevoluteJoint<double>&>(
            plant->GetJointByName("knee_joint_left")),
        0, 1500);
```
tries to cast the fixed spring joint to a `RevoluteJoint`. However, I'm not sure how much adding the suggested check really avoids errors, given that the URDF names themselves aren't completely static.

Reply:
Got it, maybe we can add a warning to the documentation?